published/009-2016-March-28.md

```diff
@@ -14,35 +14,37 @@ Package managers have been a large topic of discussion recently. Mostly, this is
 
 ### DNS outage
 
-We're using DigitalOcean for providing ipfs.io DNS. On Tuesday, March 24th, DigitalOcean DNS was hit by a [severe outage](https://www.digitalocean.com/company/blog/update-on-the-march-24-2016-dns-outage/) lasting hours, which took the public gateway at ipfs.io down. We switched to [DNSimple](https://dnsimple.com) in an ad-hoc fashion and brought ipfs.io back while DigitalOcean was still down, but this incident obviously hit us on the wrong foot a bit. We'll be working to never get taken down this way again. Here's a few things we'll do:
+We're using DigitalOcean to provide ipfs.io DNS. On Tuesday, March 24th, DigitalOcean DNS was hit by a [severe outage](https://www.digitalocean.com/company/blog/update-on-the-march-24-2016-dns-outage/) lasting hours, which took the public gateway at ipfs.io down. We switched to [DNSimple](https://dnsimple.com) in an ad-hoc fashion and brought ipfs.io back while DigitalOcean was still down, but this incident obviously hit us on the wrong foot a bit. We'll be working to never get taken down this way again. It's HARD not to depend on any single points of failure. Here's a few things we'll do:
 
-- codified DNS zones, and tools to upload them to DNS providers
-- one or two backup DNS providers
-- monitoring and failover procedures
+- codify DNS zones, and tools to upload them to DNS providers
+- keep one or two backup DNS providers
+- update our monitoring and failover procedures
 
-We'll post a more detailed postmorten on our blog in the next few days.
+We'll post a more detailed post-mortem on our blog in the next few days.
 
 ### Captain.log
 
-#### Aye, you might want to check the new js-ipfs Captain.log entry, mate!
+#### Aye, you might want to check the new js-ipfs Captain.log entry, matey!
 
 Following [js-ipfs roadmap](https://github.com/ipfs/js-ipfs/blob/master/ROADMAP.md#milestone-1---js-ipfs-on-the-browser), we’re close™ to having a workable js-ipfs version that can be used in the browser and in Node.js. This will mark a very important milestone on the IPFS project and enable a whole set of new distributed web applications to be possible. If you want to be part of this effort, check out our [Captain.log](https://github.com/ipfs/js-ipfs/issues/30#issuecomment-205884015) entry to get a full update and a list of tasks you can contribute to.
 
-### Orbit
+### [Orbit](//github.com/haadcode/orbit)
 
-@haadcode has been working on improvements to [orbit-db](https://github.com/haadcode/orbit-db), [ipfs-log](https://github.com/haadcode/ipfs-log/) and [Orbit](https://github.com/haadcode/orbit). The message history fetching is now more stable and the UI feedback for loading messages is fixed. All this work will improve the user experience of Orbit.
+![Orbit](009_orbit.png)
```

Comment:
This image of orbit does not highlight the main point of orbit: realtime chat. It's mostly hidden by the side bar. How about this one instead: 

![](https://raw.githubusercontent.com/haadcode/orbit/master/screenshots/screenshot3%202016-04-14.png)